src/sth_database.js

```diff
@@ -342,6 +374,35 @@
   }
 
   /**
+   * Saves the contents of a readable stream into a file in the local file system
+   * @param {string} attrName The name of the attribute the stream contents refers to
+   * @param {object} stream The stream
+   * @param {string} fileName The file name where the stream contents should be stored
+   */
+  function save2File(attrName, stream, fileName) {
```

Comment:
These two functions are modelic in its scope and length (related to the comments below about function length).